recipes/td2/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "td2" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Markusjsommer/TD2/archive/refs/tags/v1.0.0.tar.gz
+  sha256: b16e8040896a4aba1b5f01cd71a47b80d53e3f2ab7efdae1dc3ee52ba722c12f
+
+build:
+  noarch: python
+  entry_points:
+    - TD2 = TD2.help:main
+    - TD2.LongOrfs = TD2.LongOrfs:main
+    - TD2.Predict = TD2.Predict:main
+  script: python -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_compatible('td2', max_pin="x") }}
+
+requirements:
+  host:
+    - python >=3.9,<3.13
+    - pip
+  run:
+    - python >=3.9,<3.13
+    - pytorch >=2.1.2
+    - torchvision >=0.16.2
+    - torchaudio >=2.1.2
+    - typing_extensions >=4.9.0
+    - tqdm >=4.66.1
+    - scipy >=1.10.1
+    - numpy >=1.24.4,<2
+    - pandas >=2.0.3
+    - setuptools
+    - psauron >=1.0.5
+
+test:
+  commands:
+    - pip check
+    - TD2 --help
+    - TD2.LongOrfs --help
+    - TD2.Predict --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Markusjsommer/TD2
+  license: MIT License
```

Can you please use a SPDX license identifier?
```suggestion
  license: MIT
```